query-graphs/src/tree-rendering.ts

```diff
@@ -976,17 +976,18 @@ export function drawQueryTree(target: HTMLElement, treeData: TreeDescription) {
 
     // Add metrics card
     let treeText = "";
-    const properties = treeData.properties ?? {};
-    treeText += buildPropertyList(properties);
-    treeText += buildPropertyList({nodes: totalNodes});
-    if (crosslinks !== undefined && crosslinks.length) {
-        treeText += buildPropertyList({crosslinks: crosslinks.length});
+    treeText += buildPropertyList(treeData.properties ?? {});
+    if (DEBUG) {
```

Comment:
Is it a DEBUG situation or some other mode? Perhaps some use cases want more metrics not less. Luis?

Reply:
I agree that some use cases might want more metrics (e.g., the number of operators, an overview of which join types the query contains etc.)

However, I would argue, that the current "node count" and "crosslink count" do not provide those metrics.
E.g., on a Hyper plan, the node count includes a lot expressions, and even subparts which just happen to be separate entities in the tree view and are collapsed by default anyway.

As such, I consider the number of visible nodes mostly as an implementation detail of the query-graphs library and how it decides to render the JSON